src/common/cis2pi.jl

```diff
@@ -0,0 +1,12 @@
+"""Functions to evaluate exp(), cos(), or sin() involving 2π"""
```

Comment:
That's a bit generic. Having the formula cis2pi(x) = exp(2π i x) in the comment makes it clearer, esp since `cis` is not super standard outside julia